patches/tools-crates-gnrt-lib-crates.rs.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tools/crates/gnrt/lib/crates.rs b/tools/crates/gnrt/lib/crates.rs
+index b240dbe2fe698a7e6e86839e71ccce8048d693f4..281b2a72d0131165e2b18d41dc3f8bfd9ea90502 100644
+--- a/tools/crates/gnrt/lib/crates.rs
++++ b/tools/crates/gnrt/lib/crates.rs
+@@ -490,7 +490,7 @@ fn collect_crate_file(files: &mut CrateFiles, mode: CollectCrateFiles, filepath:
+         // md: Markdown files are commonly include!()'d into source code as docs.
+         // h: cxxbridge_cmd include!()'s its .h file into it.
+         // json: json files are include!()'d into source code in the wycheproof crate
+-        Some("md") | Some("h") | Some("json") => match mode {
++        Some("md") | Some("h") | Some("json") | Some("data") => match mode {
```

Comment:
What is this for? It needs to be submitted to upstream or use gnrt_config.toml

Reply:
removed. we'll need to address this later if we decide to bump idna

Reply:
yeah, for reference @bridiver this was a hack to make sure that the `*.data` files from `idna` were included. Since we're no longer using the newer version of `idna`, it's not needed.